topics/howtosim.rst

```diff
@@ -0,0 +1,255 @@
+
+``NO_L0`` -  turn off the monopole component (you've supplied a background model!)
+
+``NO_L1`` - turn off the dipole component
+
+``EVEN_L`` - only allow even harmonic orders to evolve
+
+``EVEN_M``  - only allow even harmonic subspaces to evolve
+
+``M0_ONLY`` - only allow the m=0 harmonic subspace to evolve
+
+``self_gravity`` - allow the component to evolve
+
+with similar options for the stellar disk.
+
+Step Four: run the simulation
+******************************
+Run your full simulation! Make an EXP YAML config file, we'll call ours "real_run_config.yml", then run it with ``mpirun -v exp --config first_step_config.yml`` . If you skipped Step 3, you'll set the disk and halo ``bodyfile`` parameters in the EXP YAML config file to the files from Step 1. If you did the full three-step relaxation process, you'll need to reformat the last phase space output from Step 3C into disk and halo body files, then you will set the ``bodyfile`` parameters to these new body files. If you generated your own disk initial conditions, the ``acyl`` and ``hcyl`` parameters should match ``ascale`` and ``hscale``, respectively, in your ``gendisk`` YAML config file (note that ``scale_length`` and ``scale_height`` can differ from ``acyl`` and ``hcyl``). For the real run of your simulation, ``self_consistent`` should be set to true for all components and ``mlim`` should be set to a value equal to or greater than your ``mmax`` parameter  (instead of setting these explicitly, these lines can be omitted from the EXP YAML config file which will set these values to the sensible defaults). 
+
+Step Three and Four can be run with a ``.sh`` script, and could even be combined into a single script. If you're using a machine with SLURM, you can set the SLURM parameters at the top of the script.
+
+Step Five: sanity check your output
+***********************************
+You've run your first simulation with EXP! You should check that the results are sensible. You can do this in a few ways, we provide a few checks that we like to perform. If you have additional checks that you do, please contribute these!
+
+#. Inspect the phase space dumps at a few times in the simulation. Do you see big rings at early times? A three armed spiral/bar? These patterns can be a sign of issues with dynamical equilibrium in the initial conditions. You can do the three step relaxation procedure to help ensure a quiet start
+
+#. Inspect ``OUTLOG file``, which should live in the same directory as your phase space dumps. Plotting column 1 versus 38 and column 1 versus 58 is a helpful diagnostic - these plots will show the virial parameter (i.e. :math:`-2T/V_c`, where :math:`V_c` is the Virial of Clausius) for the halo and disk, respectively. Perfect equilibrium would have :math:`-2T/V_c = 1` - this plot should look like tiny oscillations around one.
+
+#. Look at some orbits of star particles. Are they sensible? 
+
+
```

Comment:
An overall comment/suggestion:

Perhaps separate running `exp` from IC generation?  Two separate sections or subsections?  Clearly one needs to run `exp` to generate ICs from the 3-step process.  But, as I said earlier, some people may prefer other IC generation methods.